README.md

```diff
@@ -203,12 +231,67 @@ One good way to automate is using CloudFormation, using this example snippet:
 }
 ```
 
-#### Elastic Beanstalk Installation ####
+#### For Elastic Beanstalk ####
 
 * In order to add tags to existing Elastic Beanstalk environment, you will
   need to rebuild the environment with the spot-enabled tag. Follow this
   [guide](http://www.boringgeek.com/add-or-update-tags-on-existing-elastic-beanstalk-environments)
 
+### Configuration of autospotting ###
+#### Default configuration ####
```

Comment:
Fair point, a maybe more common way would be to use a configuration file used by the program, but I'm not sure on how possible that would be within Lambda ?